runbotics/runbotics-orchestrator-ui/src/pages/app/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const AppPage = () => {
+    return <div>Redirect to app/processes</div>;
```

Comment:
translation has to be here + it would be better to name it more user friendly like `Redirecting to page list` or smth

Reply:
User never sees that page, I'il pu loading screen here